test/e2e/common/downward_api.go

```diff
@@ -74,10 +74,20 @@ var _ = framework.KubeDescribe("Downward API", func() {
 					},
 				},
 			},
+			{
+				Name: "HOST_IP",
```

Comment:
You added a new field to an existing check without a version block check.

Reply:
ping @andrewsykim , seems you left a bug to fix here :)

Reply:
@timothysc @resouer  Thanks for bringing this to my attention :), will try to fix it this week. Is this blocking someones PR from getting in?

Reply:
Seems odd that this would come up as a problem after 6 months

Reply:
I don't think this is a blocker

Reply:
PR to fix in 1.8.1 is here - https://github.com/kubernetes/kubernetes/pull/53673

Reply:
Thanks Tim, it seems we also version blocked pod IP e2e test for > v1.8 but it was added way back, was that intentional?

Reply:
@andrewsykim I don't think we want to version block pod IP e2e test. I send PR #53772 to split the test case :)